packages/login/network/settings.js

```diff
@@ -268,13 +268,13 @@ export const getNetworksByProvider = (providerId, withLocal = false) => {
   }
 }
 
-export const getNetworksWithProviders = (withLocal = false) => {
+export const getNetworksWithProviders = (providers = [], withLocal = false) => {
   let networks = []
+  let providersArray = [...providers]
```

Comment:
Please replace let to const.

And why not just `const providersArray = providers`?